cou/apps/app.py

```diff
@@ -495,6 +496,7 @@ def _get_upgrade_current_release_packages_plan(self, parallel: bool = False) ->
             function=upgrade_packages,
             units=self.status.units.keys(),
             model=self.model,
+            packages_to_hold=self.packages_to_hold,
```

Comment:
nit: Do we really need this line?

Reply:
I don't understand. How else can we share the same `_get_upgrade_current_release_packages_plan` in subclasses when `self.packages_to_hold` actually contains something?

Reply:
Oh haha :laughing: The code collapsing in diff can be a curse sometimes.

Reply:
Same here, I was confused why it's in post_init.